Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>Nie mówiliśmy jeszcze o wyborze zmiennych, ale wybieranie ich na piechotę nie jest najlepszym pomysłem</p><p> </p><p>Skoro spodziewamy się małych błędów, to MAE może być lepsze od RMSE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>